### PR TITLE
run Prettier through ESLint

### PR DESCRIPTION
**Pull request by RedSkelly**
Removed `prettier-eslint`, added `eslint-config-prettier` and `eslint-plugin-prettier` in its place. 
Ordering in the `.eslintrc` file within `"extends":[]` is of a special order so that rule conflicts between Prettier and ESLint are nonexistent. 

In development, ESLint _should_ be running by default, as declared in the `.vscode` workspace settings. However, should it not be running, select `shift`+`command`+`P` to view the Command Palette, then search for `ESLint: Enable ESLint`. Click the command.

Future Prettier rules are to be added to `.eslintrc`. There is still a `.prettierrc` file, _only_ because without it, `stylelint` throws an error.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,20 +44,27 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "fix:code": "prettier-eslint --write 'src/**/*.{js,jsx}'",
+    "fix:code": "eslint --fix",
```
Comment (JohnathanHuggett) [inline package.json]: Why was this changed from using prettier to eslint?

File: src/components/utilities/image.js [hunk 2]
```diff
@@ -19,5 +19,8 @@ Image.propTypes = {
   className: PropTypes.string,
   imageArr: PropTypes.arrayOf(PropTypes.stings).isRequired
 }
+Image.defaultProps = {
```
Comment (JohnathanHuggett) [inline src/components/utilities/image.js]: Before we commit to this `deafultProps` pattern for the `className` prop. Let's check the browser and make sure the `class` does not compile down to `class=""`.

If it does not we are good to go. Just add a line break between `defaultProps` obj and the `propTypes` obj.

If so, lets remove this default props and add `/* eslint-disable react/require-default-props */` above the declaration of the `propTypes` to stop the error.

File: .eslintrc [hunk 3]
```diff
@@ -15,11 +20,25 @@
     "node": true
   },
   "rules": {
+    "prettier/prettier": [
```
Comment (JohnathanHuggett) [inline .eslintrc]: We need to move this back into the `.prettierrc` file. If you have an empty `rc` file it will use default settings. Then this tells it to error if this setting are not met and these setting conflict with prettiers default settings. 

Try to save a js file within this repo and you will see what I am talking about

Comment (RedSkelly) [reply .eslintrc]: I found there were conflicts due to my incorrect `fix:code` script. It is now changed, and is ran when we run `yarn run develop`. I've tested it on my end, seems to be the fix.

Comment (JohnathanHuggett) [reply .eslintrc]: That did not work for me

File: src/components/utilities/image.js (hunk 2 above)
Comment (RedSkelly) [reply src/components/utilities/image.js]: Understood, I just undid those changes for now. We can discuss later about whether we want to require default props or ignore them in the `.eslintrc`.
